src/routes/index.js

```diff
@@ -1,9 +1,11 @@
 import express from 'express';
 import user from './user';
 import auth from './auth';
+import novel from './novel';
 
 const router = express.Router();
 
 router.use('/', user, auth);
+router.use('/novels', novel);
```

Comment:
You could include the novel routes in this way `router.use('/', user, auth, novel);` so we don't have multiple `router.use` statements. Then in the novel routes file, you could then add the `/novels` prefix

Reply:
Alright, I just understood how that line worked